src/hooks/imagePicker.ts

```diff
@@ -45,3 +47,40 @@ export const useSelectImage = (
 
   return { imageUri, selectImage };
 };
+
+export const useCaptureImage = (
+  onChange?: <T>(
+    setter: React.Dispatch<React.SetStateAction<T>>
+  ) => React.Dispatch<React.SetStateAction<T>>,
+  allowsEditing?: boolean,
+  aspect?: [number, number],
+  quality?: number,
+  mediaTypes?: MediaTypeOptions
+) => {
+  const [imageUri, setImageUri] = useState<string>();
+
+  const captureImage = useCallback(async () => {
+    const { status } = await requestCameraPermissionsAsync();
+
+    if (status !== PermissionStatus.GRANTED) {
+      Alert.alert(texts.errors.image.title, texts.errors.image.body);
+      return;
+    }
+
+    // this allows for proper selecting and cropping to 1:1 images (and not videos)
+    // for more details about options see: https://docs.expo.dev/versions/latest/sdk/imagepicker/#imagepickermediatypeoptions
+    const result = await launchCameraAsync({
```

Comment:
i cannot open a camera:

> Error: Camera not available on simulator

we had working camera in simulators before, what is different now?

Reply:
I don't think there is a camera that works with the simulator. We need a real device to use the camera feature.